### DISCUSSION TITLE
is there any crontab job check server status and start it automatically?

### DISCUSSION THREAD

**Discussion by liuzibei2004** [apache/dolphinscheduler#13003] [Q&A]
when the operating system restarts，or  the server down(eg:OOM)，how to make it start automatically? 
Is there any crontab register with start command，and when execute stop command，the register canceled？

**Comment by DarkAssassinator** [comment]
hi @liuzibei2004 , u mean watchdog?

**Reply by liuzibei2004** [reply]
> hi @liuzibei2004 , u mean watchdog?

yes

**Comment by liuzibei2004** [comment]
any one knows？
